app/Services/UserService.php

```diff
@@ -23,31 +29,26 @@ public function resource(int $id, array $inputs = []): User
 
     public function update(int $id, array $inputs = []): array
     {
-        $user = User::find(auth()->id());
+        $user = $this->userObj->find(Auth::id());
```

Comment:
@dhrumin-7span Please use the Auth::user()

Reply:
Changed.